tests/comfy_cli/command/test_nodes_introspect.py

```
@@ -458,6 +458,129 @@ def test_non_string_category_does_not_crash(self, monkeypatch, capsys):
+    def test_max_paths_truncation_withholds_the_exact_claim(self, patched_loader, capsys):
+        env = _run(["path", "LATENT", "IMAGE", "--max-depth", "4", "--max-paths", "1"], capsys)["data"]
+        assert env["count"] == 1
+        assert env["truncated"] is True
+        assert env["truncated_by"] == "max_paths"
+        assert env["exact"] is False
+
+    @pytest.mark.parametrize(
+        ("flag", "value"),
+        [("--max-depth", "0"), ("--max-depth", "-1"), ("--max-paths", "0"), ("--max-paths", "-3")],
+    )
+    def test_non_positive_bounds_are_refused_not_answered(self, patched_loader, capsys, flag, value):
```

Pylint (4.0.6)

[warning] Redefining name 'patched_loader' from outer scope (line 137)

(W0621)